code/modules/station_goals/bluespace_tap.dm

```diff
@@ -250,6 +168,27 @@
 		list(1, MACH_CENTER, 1),
 		list(1, 0,		   1),
 	))
+	event_list = list(
+		new /datum/bluespace_tap_event/dirty(src),
+		new /datum/bluespace_tap_event/electric_arc(src),
+		new /datum/bluespace_tap_event/electric_arc/mass(src),
+		new /datum/bluespace_tap_event/radiation(src),
+		new /datum/bluespace_tap_event/gas/nitrogen(src),
+		new /datum/bluespace_tap_event/gas/carbon_dioxide(src),
+		new /datum/bluespace_tap_event/gas/oxygen(src),
+		new /datum/bluespace_tap_event/gas/plasma(src),
+		new /datum/bluespace_tap_event/gas/agent_b(src),
+		new /datum/bluespace_tap_event/gas/sleeping_gas(src)
+	)
+
+	radio = new(src)
+	radio.listening = FALSE
+	radio.follow_target = src
+	radio.config(list("Engineering" = 0))
+
+/obj/machinery/power/bluespace_tap/cleaning_act(mob/user, atom/cleaner, cleanspeed, text_verb, text_description, text_targetname)
+	. = ..()
+	dirty = FALSE
```

Comment:
use `post_clean` here or check if `.` is true first, otherwise you can start cleaning, walk away, and it will be clean